File: PRAKTIKUM-PEMROGRAMAN1/MODUL3/Python/PRAK305-2410817210024-MuhammadIbnuSina.py
nilai= int(input())

jam = nilai // 3600
menit = (nilai % 3600) // 60
detik = nilai % 60

if jam >= 24:
    hari = jam // 24
    jam %= 24
    print(f"{hari} hari {jam:02}:{menit:02}:{detik:02}")
else:
    print(f"{jam:02}:{menit:02}:{detik:02}")

File: PRAKTIKUM-PEMROGRAMAN1/MODUL3/C/PRAK305-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>

int main(){
    int nilai, jam, menit, detik;
    scanf("%d", &nilai);

    jam = nilai / 3600;
    menit = (nilai % 3600) / 60;
    detik = nilai % 60;

    if (jam >= 24){
        int hari = jam / 24;
        jam %= 24;
        printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);
    }else{
        printf("%02d:%02d:%02d", jam, menit, detik);
    }
    return 0;
}